src/main/java/frc/robot/sensors/ArmSensor.java

```diff
@@ -46,23 +51,24 @@ public Distance getSensorDistance() {
   public Trigger inTrough() {
     return new Trigger(
         () -> {
-          if (getSensorDistance().in(Millimeters) > TROUGH_LOWER_LIMIT
-              && getSensorDistance().in(Millimeters) < TROUGH_UPPER_LIMIT) {
+          if (getSensorDistance().in(Meters) > TROUGH_LOWER_LIMIT
+              && getSensorDistance().in(Meters) < TROUGH_UPPER_LIMIT) {
```

Comment:
Would be better to read the distance once above the `if()` statements and then compare against that, to make sure the value doesn't change between the two checks (theoretically possible).

Reply:
Ditto from below